package.json

```diff
@@ -51,7 +51,8 @@
   },
   "private": true,
   "scripts": {
-    "postinstall": "lerna bootstrap --hoist",
+    "lerna": "lerna",
+    "postinstall": "lerna bootstrap",
```

Comment:
yeah so for whatever reason this wasn't working. I added hoist to the `lerna.json` instead so it runs regardless of how bootstrap is called.